hazelcast/src/main/java/com/hazelcast/internal/config/YamlConfigSchemaValidator.java

```diff
@@ -77,8 +79,12 @@ public void validate(YamlMapping rootNode) {
                                 + " root schema document, " + definedRootNodeCount + " are present",
                         "#", "#", emptyList());
             }
-            SCHEMA.validate(YamlToJsonConverter.convert(rootNode));
+            Validator.builder()
+                    .primitiveValidationStrategy(PrimitiveValidationStrategy.LENIENT)
+                    .build()
+                    .performValidation(SCHEMA, YamlToJsonConverter.convert(rootNode));
         } catch (ValidationException e) {
+            e.printStackTrace();
```

Comment:
Probably a leftover. If you want to print the stacktrace, use the Logger instead.

Reply:
Thanks, removed.